lib/airbrake-ruby/notice.rb

```diff
@@ -19,6 +19,7 @@ class Notice
     CONTEXT = {
       os: RUBY_PLATFORM,
       language: "#{RUBY_ENGINE}/#{RUBY_VERSION}".freeze,
+      program: $PROGRAM_NAME,
```

Comment:
We can't add this property here because it's not compatible with our API. `params` or `environment` would suit more.